src/Pearify/Utils/ClassFinder.php

```diff
@@ -98,6 +104,24 @@ public function find()
         }
     }
 
+    /**
+     * Returns if given token is after a opening bracket
+     *
+     * @param $c
+     * @return bool
+     */
+    private function isAfterOpeningBracket($c)
+    {
+        for ($k = $c->from; $k >= 0; $k--) {
```

Comment:
How about type hinting `$c` & also updating doc block for `findClassInNextTokens `. It makes it easier to read & follow.

Reply:
👍